packages/core/core.scss

```diff
@@ -4,4 +4,5 @@
 @import "./headings.scss";
 @import "./paragraphs.scss";
 @import "./links.scss";
+@import "./help-text.scss";
```

Comment:
I'm thinking maybe we should put this, paragraph, and headings into one file? Maybe `typography.scss`?

If not, it should probably be called `form-support-text.scss` since we renamed the class 😊

Reply:
Good idea, will fix